src/main/java/ro/developmentfactory/thedrones/controller/dto/DroneStatusResponse.java

```diff
@@ -11,12 +12,23 @@
 @NoArgsConstructor
 @AllArgsConstructor
 @Builder
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
 public class DroneStatusResponse {
     private UUID idDroneStatus;
     private Integer currentPositionX;
     private Integer currentPositionY;
     private Direction facingDirection;
     private UUID idDrone;
+    private TurnDirection turnDirection;
```

Comment:
you don't need `TurnDirection` in the response - there's no need to know where was the turn in the response